db/migrate/20201217163340_update_enterprise_socia_links.rb

```diff
@@ -0,0 +1,18 @@
+class UpdateEnterpriseSociaLinks < ActiveRecord::Migration
+  def instagram_regex
+    %r{\A(?:https?://)?(?:www\.)?instagram\.com/([a-zA-Z0-9._@-]{1,30})/?\z}
+  end
```

Comment:
You already have this method within the Enterprise model, Why not to call
`enterprise. instagram_regex` instead?

Reply:
make sense, I'll do it in that way! Thanks :)